cross/Dockerfile

```diff
@@ -88,6 +88,22 @@ RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add - && \
     DEBIAN_FRONTEND=noninteractive apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
     rm -f /etc/apt/sources.list.d/yarn.list
 
+# ------------------------------------------------------------------------------------------------
+# install kind
+RUN curl -Lo /usr/local/bin/kind https://github.com/kubernetes-sigs/kind/releases/download/v0.4.0/kind-linux-amd64 -s &&\
+    chmod +x /usr/local/bin/kind
+
+# install kubectl 
+RUN curl -Lo /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.15.0/bin/linux/amd64/kubectl -s &&\
+    chmod +x /usr/local/bin/kubectl
+
+# install helm
+RUN cd /tmp &&\
```

Comment:
Helm is already installed through our make files: https://github.com/upbound/build/blob/master/makelib/helm.mk#L150 

Do we need to install it here too?

Reply:
I noticed that too. However, it seems that this always downloads helm to the src directory and doesn't take advantage of docker cached image. Is this intended?
If we decided to keep helm as is, shouldn't we also install other tools similarly?

Reply:
removed it from the cross image